src/HornedBeasts.js

```diff
@@ -1,45 +1,52 @@
-import React, { Component } from 'react'
-import Card from 'react-bootstrap/Card'
+import Button from "react-bootstrap//Button";
+import React, { Component } from "react";
+import { Container } from "react-bootstrap";

```

Comment:
Hi there! I believe this could be throwing the error you and I spoke about on Canvas (Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined.)

I checked Bootstrap and this line should read `import Container from  'react-bootstrap';`  

I am hoping this will do the trick. If not, a TA can for sure help with netlify and I will continue to work on it too 👍